### ISSUE TITLE
Request: Make character names a different color than text

### ISSUE THREAD

**Issue by Raisoshi** [Yenof/chat-panel#1]
This is a really great plugin and I like it a lot, but when looking at my clan chat it's just a jumble of words at first glance. If we could change character name to be a different color than text it would be a lot easier to recognize different message lines from each other.

I haven't tested much or am that interested in the other chat panels, but something similar could probably be done and would be great, maybe add an option for a chat timestamp like we can have for the in screen chat (not sure if a plugin or baseline) and make that a different color?

**Comment by Yenof** [comment]
Hi, thank you very much for the brilliant suggestion. 
I am now actively looking into adding Time Stamps, and configurable Time stamp and Name color. 
Thanks again! :D

**Comment by DesignSir** [comment]
Adding here, instead of new issue
Having it slightly alternate colour each message would also help/ be awesome or even a blank line between messages, so its easier to digest

**Comment by Yenof** [comment]
Update on Time Stamps, it's now live in 1.1.

And I have line spacing working in upcoming 1.2.

Really great additions, thank you both. :D

**Comment by Yenof** [comment]
This is not complete or fully functional, but I finally have some progress to show for the name recoloring. :D

![image](https://github.com/Yenof/chat-panel/assets/122739279/dbd65f4c-a6c2-4d39-83ec-c09972253930)

I had to change some seemingly vital stuff to get this as functional as it is, so I want to test it for a really long time before I make it live. Hopefully it will be worth the wait. :D

**Comment by DesignSir** [comment]
Man, that is looking awesome dude, i already use this during raids - once this is ready it will be dope

**Comment by Yenof** [comment]
Name coloring is live in [v1.6](https://github.com/runelite/plugin-hub/pull/5599). 
I think everything works well. :D

@DesignSir With line spacing and name coloring now added, would you still like to have alternate color each message option?

**Comment by Yenof** [comment]
![image](https://github.com/Yenof/chat-panel/assets/122739279/56435343-f9d1-49a8-ae20-381cfb03bd48)

Hi, I finally have an update on 2 of the other features mentioned here.

"Odd Row Shading" will be able to tint alternating rows of chat. It can go from -100 to 100, with negatives being darker, and positive making it lighter. 

"Timestamp Color" and it's companion "Enable Custom Timestamp Color" let the timestamp color be overridden for all chat tabs. 
I debated making the Timestamp color options per chat tab, but landed on 1 config for all tabs for now. 


Both of these features seem functional at the moment, and I plan to have them added with the next update (1.9).

**Comment by DesignSir** [comment]
Apolgoies I didnt respond, I missed the notifcation somehow

spacing & timestamp have made it really easy to read, this is going to be even better, thank you for working on it :)

**Comment by Yenof** [comment]
I have mixed emotions finally closing my first plugin's first issue, but I'm very glad to have gotten these great suggestions so early into Chat Panel's existence. 

Who knows if or when these features would have ever come to Chat Panel without your contributions; thank you both for your help!

:)

**Comment by DesignSir** [comment]
Thanks for all the work my man, I use your plugin all the time
